x-pack/plugins/stack_connectors/public/connector_types/slack_api/slack_api.tsx

```diff
@@ -29,6 +30,20 @@ export const getConnectorType = (): ConnectorTypeModel<
   PostMessageParams
 > => ({
   id: SLACK_API_CONNECTOR_ID,
+  group: [
```

Comment:
what about `subtype` instead of `group`?

Reply:
done